mappings/net/minecraft/block/material/Material.mapping

```diff
@@ -23,11 +23,11 @@ CLASS net/minecraft/class_63 net/minecraft/block/material/Material
 	FIELD field_131 FOILAGE Lnet/minecraft/class_63;
 	FIELD field_132 PLANT Lnet/minecraft/class_63;
 	FIELD field_133 REPLACEABLE_PLANT Lnet/minecraft/class_63;
-	FIELD field_134 CHEESE Lnet/minecraft/class_63;
+	FIELD field_134 SPONGE Lnet/minecraft/class_63;
```

Comment:
cheese

Reply:
Gonna ask: why `cheese`?

Reply:
Also, was already approved here: https://github.com/Legacy-Fabric/yarn/pull/255/files#

The only reason it's popping up again is cuz of the squash merging, and I had already started the `client` stuff before rebasing my fork.